app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -239,6 +299,21 @@ export default class UserMenu extends Component {
     ];
   }
 
+  get _assignedNotificationTypes() {
+    return this._topTabs
+      .concat(this._bottomTabs)
+      .filter((tab) => tab.notificationTypes)
+      .map((tab) => tab.notificationTypes)
+      .flat();
+  }
+
+  get _unassignedNotificationTypes() {
```

Comment:
"assign" makes my brain go to the assign plugin right away, can we pick a different name please 😅? Maybe `notificationTypesForTheOtherTab`?